types/api.go

```diff
@@ -475,7 +476,7 @@ type TokenRequest struct {
 	RefreshToken string `json:"refresh_token,omitempty"`
 
 	// Code and CodeVerifier are required if GrantType is 'pkce'.
-	Code string `json:"code,omitempty"`
+	Code string `json:"auth_code,omitempty"`
```

Comment:
Sorry, this one was addressed in #6, so there's a conflict here now - you'll need to rebase this PR

Reply:
Done!